### PR TITLE
CSSWG update

### PR DESCRIPTION
**Pull request by mirisuzanne**
red panda face, licking it's lips

## Related Issue(s)
_Reminder to add related issue(s), if available._

- #666 

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by mirisuzanne** [comment]
Still needs an image, but I think everything else is there. I'd aim to publish on Tuesday, 23rd.

File: content/blog/2024/csswg-july.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
+---
+title: Working Group updates for June/July
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-july.md:2]:
```suggestion
title: CSS Working Group updates for June–July
```

**Review by jgerigmeyer — APPROVED** [review]

File: content/blog/2024/csswg-july.md (hunk 1 above)
Comment (jgerigmeyer) [reply content/blog/2024/csswg-july.md:2]: Or:

```suggestion
title: CSS Working Group updates for June & July
```

**Review by SondraE — APPROVED** [review]
- The _by Miriam Suzanne_ line appears to be missing. 
- Make the _u_ in _updates_ uppercase.
- "Later this month, we’ll be doing a live stream with Stephanie Eckles..." Here _live stream_ could link to https://www.youtube.com/watch?v=aDMWD_CYpEI
- In the Wide-Gamut Colors section, is there a place to link to the Winging it about colors? https://www.youtube.com/watch?v=Lq4saw4Rqe0
- The _g_ and _c_ in _Wide-Gamut Colors_ subtitle should be uppercase to maintain capitalization consistency within this article.

File: content/blog/2024/csswg-july.md [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+  (hybrid online/in-person) throughout the year,
+  and they always result in a flurry of activity!
+  Here's a rundown of some highlights
+  from the last few months,
+  with a focus on the features I maintain.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+{% callout %}
```
Comment (stacyk) [inline content/blog/2024/csswg-july.md:34]: great callout

File: content/blog/2024/csswg-july.md [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+our open-source work around CSS,
+consider becoming a [GitHub sponsor](https://github.com/sponsors/oddbird),
+or contributing to our
+[Open Collective](https://opencollective.com/oddbird-open-source).
+{% endcallout %}
+
+## Container Queries & Units
+
+spec:
+~~[Containment, Level 3](https://www.w3.org/TR/css-contain-3/)~~
```
Comment (stacyk) [inline content/blog/2024/csswg-july.md:50]: on hover, the strikethrough also grows, lol. I didn't think that would happen.

**Comment by mirisuzanne** [comment]
@SondraE Since this post talks about the live stream, should I change the date and post it today?

File: content/blog/2024/csswg-july.md [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+that behavior is designed to avoid loop conditions.
+Elements sized with viewport units _can cause scrollbars_,
+which would change the size of the unit,
+potentially removing the need for scrollbars.
+But we don't have that issue when scrollbar spacing is stable --
+using `overflow: scroll` or
+[`scrollbar-gutter: stable`](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter) --
+so the working group resolved to loosen the restriction.
+
+Moving forward, [*stable* scrollbars](https://github.com/w3c/csswg-drafts/issues/6026)
```
Comment (stacyk) [inline content/blog/2024/csswg-july.md:116]: yay!

File: content/blog/2024/csswg-july.md [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+is now eating it's own tail,
+but it seems like we have three basic approaches
+to choose from
+(and then small variations on each).
+I have a clear favorite there,
+but feel free to leave your thoughts as well.
+
+If possible,
+try not to get lost in alternative naming details --
+and focus first on the overall behavior of the feature!
```
Comment (stacyk) [inline content/blog/2024/csswg-july.md:275]: Good advice 🤣

**Review by stacyk — APPROVED** [review]
What a big list of updates! Love that so much is happening and it is nice to see what is being worked on if people have opinions.

File: content/blog/2024/csswg-july.md (hunk 3 above)
Comment (mirisuzanne) [reply content/blog/2024/csswg-july.md:50]: whoa, interesting. Seems to be Safari only.

**Pull request merged**